WBooks/BookDetailViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  BookDetailViewModel.swift
+//  WBooks
+//
+//  Created by Matías David Schwalb on 30/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class BookDetailViewModel {
+
+    var bookID: Int
+    
+    init(bookID: Int) {
+        self.bookID = bookID-1
```

Comment:
Why the `-1`?

Reply:
Data stored in BookDB.bookArrayDB is sorted using bookID in ascending order, so we can access a certain book just using the bookID as an index. For example, if we want to access a book with bookID = 3 we would have to do it like so: "BookDB.bookArrayDB[3]", but as Swift indexes from 0, we actually have to substract one to access it through index: "BookDB.bookArrayDB[2]" actually accesses the book with bookID == 3

Reply:
Okay, in that case, it should be in the corresponding View Model. Let's leave it like that for the time being. We'll do it once we get to the Refactor card.